py/tests/encoders/date_test.py

```diff
@@ -78,6 +78,8 @@ def testDayOfWeek(self):
     # 2010, 14:55
     d = datetime.datetime(2010, 11, 4, 14, 55)
     # d = datetime.datetime(2010, 11, 1, 8, 55) # DEBUG
+    # Any Monday morning (before noon) # DEBUG
+    # d = datetime.strptime("22/07/19 8:00", "%d/%m/%y %H:%M") # DEBUG
```

Comment:
Thanks @shahardror ! :+1: 
I'll merge this so we can work with it in #582 
we'll then want to : 
- add this after the existing `d= datetime...`
- uncomment it so it triggers the failure
- add a try-catch to validate the `encode` does not fail (c++ gtest has EXPECT_NO_THROW, i think py assert does not, so we'd have to do...) 
```
try: 
  bits = enc.encode(d)
except:
  raise "Encoding before Monday noon failed!"
```